src/tools/fuzzing/fuzzing.cpp

```diff
@@ -1648,6 +1648,20 @@ void TranslateToFuzzReader::modifyInitialFunctions() {
       //       them.
     }
   }
+
+  // Add invocations, which can help execute the code here even if the function
+  // was not exported (or was exported but with a signature that traps
+  // immediately, like receiving a non-nullable ref, that the fuzzer can't
+  // provide from JS). Note we need to use a temp vector for iteration, as
+  // addInvocations modifies wasm.functions.
+  std::vector<Function*> funcs;
+  for (auto& func : wasm.functions) {
+    funcs.push_back(func.get());
+  }
+  for (auto* func : funcs) {
+    addInvocations(func);
```

Comment:
Should we do this with some probability in case the initial contents has a function that always traps or something like that?

Reply:
`addInvocations` already does nothing 50% of the time.

Reply:
If there are lots of functions in the initial contents and they all trap, that still might be an issue, so maybe we want to skip this entirely half the time.

Reply:
True, though it isn't that harmful: if an invoke traps, that just means we see
```
fuzz-exec calling foo
[trap]
```
in the output. But those two lines don't affect any other code. The only cost is the time to run them, I suppose, or did you have something else in mind?

IOW from my perspective it is important to avoid adding traps to code (when adding random code), but adding an entire new export that traps is not a big deal.

Reply:
Ok, I was thinking that traps would prevent subsequent code from running. Since that's not the case, LGTM as-is. Thanks!